### ISSUE TITLE
Directory restructure

### ISSUE THREAD

**Issue by Cyberzoid1** [uidaho/smartsettia-unit#8]
I'm considering moving all my secondary python files to a library folder.
This will move much of the bulk out of the main directory to leave it a bit cleaner.
Slight technical issue when it comes to importing those files. But I found a way.

1) Is my current working directory. There are a few cache files and temp files but as you can see it is more cluttered.

2) Is the proposed structure
Mainly thinking on a end user standpoint.

![image](https://user-images.githubusercontent.com/18319510/32577143-cb79f46c-c48d-11e7-8fdb-79350a66a2a9.png)

Opinions?

**Comment by brec9824** [comment]
If it isn't going to cause any issues for you I think 2 looks better from a user perspective.

**Comment by brandonjank** [comment]
#2 looks better. I suggest that you follow the file structure style guide created by the makers of python at: http://docs.python-guide.org/en/latest/writing/structure/

Notice the folder name for the module is "sample", in our case it would be "smartsettia", "smartsettia-unit", or perhaps "unit". For example:
```
README.rst
LICENSE
unit.py
setup.py
requirements.txt
unit/__init__.py
unit/core.py
unit/helpers.py
docs/conf.py
docs/index.rst
tests/test_basic.py
tests/test_advanced.py
```

An example github from a python dev can be found at: https://github.com/kennethreitz/samplemod

**Comment by Cyberzoid1** [comment]
Implemented with commit c151b6f7fa5ccffa37d77158bff850b5a8136bf7
